### DISCUSSION TITLE
Initial seeding of <podcast:license> tag values - optimization

### DISCUSSION THREAD

**Discussion by Dwev** [Podcastindex-org/podcast-namespace#603] [Enhancement Proposal]
I’m a podcaster, and a contributor to the open source Castopod hosting service, and I’m looking into adding specific support for the `<podcast:license>` tag (e.g. in the UI with a drop-down list of licenses, as opposed to a freeform tag in the “additional RSS” field Castopod uses.

The SPDX list that is referred to on the tag information in the spec is very long, and includes licenses that have no relevance to podcasts (_Adobe Utopia Font license?_) so I’m wondering if there is a smarter way to approach this.

Ideally, I'd like to present the podcaster with a list of the most common licenses used for them to select from.  In my mind that's likely to be _All Rights Reserved_ and the _Creative Commons_ variants, but there could be more. We could then iterate on this list if there are other licenses being used that aren't in the "pre-selection".

Right now, the podcastindex API doesn't allow me to filter podcast feeds by the license tag, so I can't get an idea of how it's currently being used, even in aggregate.

I believe it would be be useful to the community to see this kind of aggregate data on certain tags in any case - to make it easier to support Podcasting 2.0.

So for @daveajones, @benjaminbellamy, @jamescridland, @samsethi, and others - do you have a feel for the types of content licenses people are using for their podcasts, or access to data in aggregate that will give us a data-backed starting point?

**Comment by daveajones** [comment]
I'm very much in support of people in the 2.0 community who are good on the license/legal front helping pare down the giant list into only what's practical.  We can then publish that as the recommended list in the main namespace document and be open to adding to it in the future upon request/consensus.

I'm actually going to put this into Phase 7 plan.  It's something I know @jamescridland has wanted.

**Comment by samsethi** [comment]
In TrueFans we had the exact same problem as @Dwev - we looked at the 40+ licenses in the SPDX list.  Originally we listed them all in a drop down but then we decided less is more and now we only list the 6 Creative Commons licenses.  - see image 1 below.  - we store this in the License Tag.  

We then display this license on the front-end app in the podcaster's info/about page with a link back to the Creative Commons page for more detail about the license.  - see image 2 below

We are currently talking with Julie/Jim Costello about adding a new music license based on Creative Commons.  It would be a V4V license with similar attributes to the CC license such as share-alike or non-commercial.  The one difference is the license will be revokable and it will have a few more attributes directly related to music.  

@joksas from RSSBlue will probably be the first host to add it and we will then add it to the list of licenses we currently support.  There is nothing to say we can't extend this list with other licenses in the future but for now I think we are covered with CC licenses. 

Personally, I think ALL podcasters need a license to let people know WHAT they can do with their content because with Remote Items, GIUD's and VTS it is now much easier to include audio content from other podcasts or music and pay the other content owner.

However not all content owners will want their audio reused and a license makes it clear they permit or do not want their music used in a musicast or a podcaster may not allow their podcast to be clipped.  

In the future, like the block and lock tags, apps will read the license and if the content license prevents it reuse then we could grey out the clipping option or prevent it being added to our VTS admin user interface.  

But to simplify, we have just used the 6 CC licenses in a drop down or we read the license tag.  The other thing we do is auto-populate the copyright field if this is blank.  

**Image 1- TrueFans Admin Backend**
<img width="795" alt="Fullscreen_01_02_2024__17_25" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/30f7da4a-7988-4f61-870e-a38c4665c322">

**Image 2- TrueFans User Frontend**
<img width="1440" alt="Fullscreen_01_02_2024__17_32" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/50e83196-b62e-43f0-ab8c-e442da0af084">

**Reply by Dwev** [reply]
Yes, I noticed how you did it at Truefans, @samsethi - it prompted me to add that and the update frequency tags to my feed manually ;)

How do you manage when a podcaster wants to reserve All Rights and not release under CC?  I noticed it was either CC or "None" in the UI.

The initial "top" list I was thinking was the 4.0 CC licenses, ARR, and PD.  **None** would be the default for most, and hosts can encourage podcasters to then select a license (giving good explanations in documentation somewhere - since this is effectively a legal instrument.)

- **None**/Undefined 
- **All Rights Reserved**
- **CC-BY-4.0** (Creative Commons Attribution)
- **CC-BY-NC-4.0**
- **CC-BY-NC-ND-4.0**
- **CC-BY-NC-SA-4.0**
- **CC-BY-ND-4.0**
- **CC-BY-SA-4.0**
- **CC0-1.0**

Apart from these, are there others that make sense?

**Reply by samsethi** [reply]
Thanks Guy. We will add ARR. What is PD and why did you include CC0-1.0 - was this the original universal license? 

What we do is look for is the license field in the RSS Field and overwrite anything stored locally.  The default is blank/none and in the user onboarding we will now add a prompt to the user.  

Happy to add other licenses as appropriate ie the new V4V music license.

**Reply by Dwev** [reply]
Sorry, **PD** = **Public Domain** which I'm pretty sure = **CC0**. It's an explicit way of saying "No Rights Reserved" and dedicating all those rights to the Public Domain.

Where can I find more information on the V4V Music license?

**Reply by jamescridland** [reply]
@Dwev - Could I challenge the "Undefined"? [All material is copyright automatically as soon as it is created](https://www.copyright.gov/what-is-copyright/) (in all jurisdictions) - so the default is automatically "ARR", and there legally can't be a "none", can there?

**Reply by Dwev** [reply]
@jamescridland Copyright and License are different things though.
The "undefined" is a marker to show a license hasn't been selected, so it can be clear to the listener that licensing is not defined and should they wish to use the content, they should find out more.
I'm not sure you could assume that a podcaster who hasn't explicitly selected a license intends to reserve all their rights - but I could be wrong.

**Reply by jamescridland** [reply]
> The "undefined" is a marker to show a license hasn't been selected, so

So it doesn't have a licence available, and therefore is fully copyright; and the default is automatically "ARR"?

**Reply by Dwev** [reply]
I think the issue for me is that I don't want to make assumptions on behalf of a podcaster, with an explicit declaration that all rights are reserved.  
In some interpretations, the lack of a declared license means that some rights may revert to the T&Cs of a host, where they may declare that some rights are not reserved by the podcaster.  Think of some of the social media sites where there is an automatic license for them to use your data / photos etc.
Maybe the term is better as "**undeclared**" instead of "**undefined**"?  Then hosts can highlight to their users why choosing a license is important, and explain the differences between them, and then the user is making a _conscious (and better educated) choice_.
To me, **undeclared** is a signal of "not known - seek more information on how this can be used", not an implicit choice.

**Comment by daveajones** [comment]
I'll start building a JSON list for the repo based on this:

- None/Undefined
- All Rights Reserved
- CC-BY-4.0 (Creative Commons Attribution)
- CC-BY-NC-4.0
- CC-BY-NC-ND-4.0
- CC-BY-NC-SA-4.0
- CC-BY-ND-4.0
- CC-BY-SA-4.0
- CC0-1.0

**Reply by Phantom-Power-Music-JC** [reply]
HI! It's me. 👋🏻 I'm super nervous about joining GitHub. Feeling like an imposter here. 

Attempting to help with a blanket license for V4V that would cover both music and podcasting, operating under the assumption that a podcast or an audio file are both either/or a "Sound Recording" or "Video Recording." 

The Draft (4.0) can be found here but still needs community wide and legal input on language:

https://www.phantompowermusic.io/phantom-power-music-license

The current suggested notation for V4V is as follows, recognizing that the technical ability for an app to include these notations, specifically the ISRC code, are still under discussion. 

©2023-V4V-ARR-CV4V-ND/S-FR-RP(USA)-QZTAX2254590

Defined As: 

©xxxx - Copyright date noting the year of first publication

V4V - Signifies the presence of Value-4-Value enabled feature within “App”.

ARR - All rights reserved by Owner(s).

C4V4 - Commercial use allowed only when an obvious exchange of value is present and an instant exchange of value is available by and between the “Owner” and “Listener.” Service provider(s) are required to make it clear to listeners that their app is not available for commercial uses such as malls, restaurants, or other public spaces without the CLEAR and AVAILABLE transfer of value explained and displayed. **This notation is important to protect podcasters when using music files within a podcast as a technically commercial use.

NDS - No derivatives of work or synchronization shall be used without the express written permission of “Owner”. **This assumes the use of the song in its entirety, even with some talking on top of or underneath, is not considered a derivative work. 

FR - All compositions and licenses are fully revokable at any time by request of  “Owner.”

RP(USA) - Rights protection will be governable by country of origin of upload “Owner.”

ISRC - ISRC Code must be attached to every license notation

Julie

**Reply by jamescridland** [reply]
Welcome, Julie! One tedious boring comment:

>RP(USA)

As I understand it, the ISRC [always includes the country registration authority](https://usisrc.org/about/constructing_with_code.html) (QZ in this case = USA). I'm wondering whether this is explicitly required, therefore - or whether the ISRC does this anyway.

If it's really needed here (is it possible to be different to the one in the ISRC?) then could I recommend we use the standard two-letter abbreviations for countries? US/CA/BG/FR/DE etc?

**Reply by Phantom-Power-Music-JC** [reply]
James, interesting point. Seems to me that could be implied - IF - the ISRC code is incorporated into the final license and then transmitted from the host to the apps to display the full ISRC Code. We've been through several iterations of this language already and one attorney had suggested that we should identify the country of origin for future legal issues, which will undoubtedly surface. I think the standard two letters is fine and it seems logical to me that one would suffice to mark the country of origin. It just depends upon how this actually gets implemented. 

I do believe there is still some discussion amongst developers about why, or why not, to include the ISRC as part of the actual license.  I feel very strongly that - for music - the ISRC code should be included, if not in the license notation itself, in the Feed. The entire music industry operates around ISRC codes and having this included will help us validate copyright dates and ownership claims that come in the future. 

This document was originally written to be VERY specific to music and has since been generalized to include V4V as a whole. So please let me know if there is anything else that would help podcasting as that is not my area of expertise. Adam made the argument that a podcast and an audio file are technically the same thing, which resulted in the terminology "Sound Recording" or "Video Recording" vs. "Master Recording."  

I want to add the disclaimer that I am not an attorney in any way, shape or form. :)  But I am consulting with several and depending on how this gets implemented I will make sure to share feedback. I've made note of the country of origin, duplication. Thank you!

**Reply by Dwev** [reply]
Sorry if this is a dumb question, I'm a bit new to V4V and I am definitely not a lawyer.

How does the presence of V4V impact the rights/license assignment? Does it mean if content is sponsored or ad-supported, we would change V4V for "AD" or "SPON" or similar?

**Reply by Phantom-Power-Music-JC** [reply]
Hi Dwev, A little fuzzy on this question since I'm not on the podcast side.  Adam or Dave may be better to address this question. 

This license was created for the singular use of content on a Value4Value platform only, and where, presumably, there is not any ad supported agreements. If one were to choose this license it would indicate that in order for a platform to use it, there must be an exchange of value available between the listener and the owner of the sound or video recording.  We would not want this content ingested by a non - v4v platform. Obviously, there will be a learning curve on this for any non-v4v apps that are reading the index now.

If the sound recording - in this case presumably a podcast with ad support is played in the V4V app then I am not sure how or what legal obligation you might have to your ad partner regarding the collection of bitcoin as a payment. I am also not a lawyer and approached our conversations from the perspective of music. I had originally envisioned two variations of the license one for music and one for podcasts but Adam felt like we could change the definitions and it could cover both.  I would highly recommend someone in podcasting chime in about the use of this license when there are also ad relationships involved. 

Does this help? Happy to keep digging away at this and if we need to add language that might help podcasting I am happy to do so.

**Reply by joksas** [reply]
Had a chat with Julie and Jim re V4V license, below are some of the possible changes after others' feedback.

## License name

This license is already closely tied with the concept of an RSS feed, e.g., license mentions how value blocks live in the feed. Because of this, other creator-specific information may live in the feed too.

For example, instead of `2023-V4V-ARR-CV4V-ND/S-FR-RP(USA)-QZTAX2254590`, the name of the license might simply be `V4V-ARR-CV4V-ND/S-FR`, while information about the year, country, and ISRC may live in the feed. At RSS Blue, we already allow the artists to include ISRC in an experimental `<podcast:isrc>` tag: <https://feeds.rssblue.com/millennium>.

One of the main goals of the license would be to make the RSS feed the source of truth in the legal sense. This would also prevent us from encoding user-specific details inside the license text or name.

## ISRC requirement

A hosting company like RSS Blue can buy lots of ISRCs cheaply, but for self-hosted creators or those who don't care about ISRC, making it a requirement may not make sense. Therefore, we are thinking of making ISRC optional.

To be clear, for music artists, it's absolutely recommended to include ISRC for every one of their tracks. In fact, many hosting companies might even make it a requirement to reduce future legal risk. However, if an individual self-hosted creator wishes to not associate their audio recording with an ISRC, they should have that option and accept the risk.

## App fees

@adamc199 and @daveajones seem to dislike the idea of maximum app fees. Full disclosure—this was my suggestion, so I'm to blame 😂

Anyway, we still think that free market may not always determine what's the best app for the podcasters, only what's the best app for the listeners. An app taking a large split may not always be obvious to the listener or even be their main priority.

Instead of maximum app fees, we feel the following two things would be perfectly reasonable to include in the license:
1. For accounting purposes, the apps should be required to include certain information in the TLV records. Specifically, a) app name, b) feed and item identifiers, c) the total amount sent by the listener, d) the value amount taken by the app. Most apps already send a, b, and c, but there's nothing requiring them (or any new apps that appear) to actually do that, so we believe it's important to make this requirement explicit. d would make it transparent how much in fees the apps are taking.
2. Feed owner should be able to block specific apps from ingesting the media in the feed; this information would be embedded in the feed itself. In practice, we already have a mechanism for that—`<podcast:block>` tag.

These two requirements enable creators to 1) observe whether the fees that apps take are fair and 2) gives the ability to block individual apps that they believe are not playing fairly. We're not inventing anything new here, just incorporating existing concepts into the license and giving more power to the creator.

**Reply by samsethi** [reply]
_"the name of the license might simply be V4V-ARR-CV4V-ND/S-FR, while information about the year, country, and ISRC may live in the feed."_ 

- 100% agree a simplified license broken up inside the feed is a great way to go. 

- We already support **YEAR** and **ISRC**.  <podcast:isrc> - RSSBlue and PodHome support this.  We show the ISRC code in the UI when we detect medium=music. This is not part of the current P2.0 spec. Should this tag be added?  

- We added a NEW **COUNTRY** tag a year ago for another feature we are building. This is not part of the current P2.0 spec. Should this tag be added?  

<img width="750" alt="TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/1a51b5d4-15d0-43dc-adc0-4459fbc3750d">

Having all the component parts would allow us to display the full license on the podcast info page and link this back to license owner e.g Phantom Power Music.  

As you can see below we already show the country/flag, the copyright, we show the producer, the host and if they support V4V.  We show a license and a link. So for this example, we would show V4V-ARR-CV4V-ND/S-FR on this page 

**So the ask is can we formally add the country and ISRC tags?** #554 - Phase7

<img width="883" alt="Notification_Center" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/35e7ffd0-c4e7-4761-a3b5-a9ee70eb45b1">

**Reply by jamescridland** [reply]
> can we formally add the country and ISRC tags

Again, just for noting, the country is part of the ISRC tag. Those working on this should work out what happens if someone says they want this in GB but the ISRC tag says it's in US.

**Reply by agates** [reply]
> apps should be required

I would recommend to stop using phrases like this or you're going to be disappointed very quickly.  There is and never will be any way to enforce it.

**Reply by samsethi** [reply]
As an app provider I agree there is a need for transparency but I am not required to do it. We are putting in options for users to choose what they want published in their TLV record.

**Reply by Dwev** [reply]
> Those working on this should work out what happens if someone says they want this in GB but the ISRC tag says it's in US.

ISRC country is just where the artist decided to register the track for legal purposes. 
Feed country is where a feed owner wants to identify where the feed is based.  

I think these are different things and can be different values. Technically, each track on an album could have a different ISRC country.

Also ISRC only applies to music as far as I can see, and a traditional podcaster may still want to identify a production country (separate from the location, which can change).

**Reply by Phantom-Power-Music-JC** [reply]
Interesting catch @Dwev I think that could be true. Different country of origin for Legal Rights Protection, vs. I'm living in England and am distributing here this year and the ISRC Code I have was generated in London.  It does appear to me that ISRC is for music only, I have not heard a podcaster have any interest using the ISRC yet. I can't help but wonder if we should still consider two variations of a V4V license. One for Music and One for Podcasting as the ISRC is very important to music and I would almost always recommend an artist get one, even a brand new fully independent artist and really should be "required" at upload for tracking the sound recording and legal copyright protections (only in terms of when it first was distributed.)

**Comment by daveajones** [comment]
Ok, started the list.  How does this look?  This is just a start, to see if this format will work.

https://github.com/Podcastindex-org/podcast-namespace/blob/main/licenses.json

**Reply by Marzal** [reply]
The format looks good, but I would use the Canonical URL 
https://creativecommons.org/licenses/by-nc/4.0/ instead of https://creativecommons.org/licenses/by-nc/4.0/deed.en

And I think would be a good idea to put the licenses in order by openness, in case the apps list them as they are in the json.
It would help get a sense of how they work.

* CC0-1.0
* CC-BY-4.0 (Creative Commons Attribution)
* CC-BY-SA-4.0
* CC-BY-ND-4.0
* CC-BY-NC-4.0
* CC-BY-NC-SA-4.0
* CC-BY-NC-ND-4.0
* All Rights Reserved
* None/Undefined

**Reply by Dwev** [reply]
I would put them in the reverse order, in descending openness, as this feels it is more protective of the copyright holder's rights, in that they have to make more effort to give away more rights.

There should probably be a strong recommendation that implementation should come with a "custom" option as well, in case there is licensed content that isn't covered by any of the ones in the list.

**Comment by Phantom-Power-Music-JC** [comment]
@joksas Not sure how to add someone to the conversation.

**Reply by joksas** [reply]
All good, this repository is public so I can see it :)

Later today, I'll summarize our discussion for others.

**Comment by dhk2** [comment]
Nothing in the V4V process requires retaining rights in a corrupt and broken system, and to do so just leads to revokation issues down the road.

There should be a released into the public domain option.

**Reply by Phantom-Power-Music-JC** [reply]
@dhk2 I agree that if an artist wants to fully release "at their own risk" in public domain, they should be free to do that. Just want to make the distinction here that we are in no way attempting to make this license something that works within the existing system. This license is an agreement generated by the artist to allow the song to be played in a very limited capacity within the "valueverse", on a "value cast" and in partnership with a "value host" and only when value-4-value features are enabled. For someone that is "professionally aspiring" this is important since we don't want a hit song scraped from the "Valueverse" and put into a huge blockbuster movie without some renumeration. Since there is no label or collection society involved (at least for now) this all happens outside of the mainstream music industry. Independence does not negate the need for an artist to protect their original work or to make licensing available for limited purposes (if so desired).  Folks often see "license" and equate this with trying to fit into the music industry. That could not be further from what we are attempting to do here. As the owner of the work you should have the right to distribute freely, or with limitations, if so desired. **Not an attorney either... so just trying to take in everyone's ideas to make the best license we can for independent users.

**Reply by adamc199** [reply]
I recommend we use language that makes this a podcast license. Not a music license or "value cast" license. 

Feeds with mp3's are podcasts. Regardless of content or length. 

The medium tag describes type of content. 

This is a copyright license that should describe use for inclusion as a remote item in another medium type.
